images/build/debian-base/variants.yaml

```diff
@@ -1,5 +1,4 @@
 variants:
-  # Debian 11 - Kubernetes 1.23 and newer
-  bullseye:
-    CONFIG: 'bullseye'
-    IMAGE_VERSION: 'bullseye-v1.4.3'
+  bookworm:
+    CONFIG: 'bookworm'
+    IMAGE_VERSION: 'bookworm-v1.4.3'
```

Comment:
@kubernetes/release-engineering are we going back to v1.0.0 with a new distro or do we keep the version?